src/NHSD.BuyingCatalogue.Ordering.Api/Models/StatusModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class StatusModel
+    {
+        [Required(ErrorMessage = "StatusRequired")]
+        [RegularExpression("Complete|Incomplete", ErrorMessage = "InvalidStatus")]
+        public string Status { get; set; }
```

Comment:
Can we use an `enum` here or does it cause validation problems?

Reply:
Validation problems all the way